src/partials/_head.hbs

```diff
@@ -2,6 +2,8 @@
 <meta charset="UTF-8">
 <meta name="author" content="Andrew Spencer">
 <meta name="viewport" content="width=device-width,initial-scale=1.0">
+<link rel="shortcut icon" type="image/png" href="images/AME_favicon.png">
+<link rel="icon" type="image/png" href="images/AME_favicon.png">
 
```

Comment:
This is a nice addition. This whole favicon thing is really painful, though. If you haven't already, check out https://github.com/joshbuchea/HEAD#favicons and the links there. That entire README has about all you could want to know about what goes inside `<head>`